tests/transactional/install_updates.pm

```diff
@@ -39,7 +39,8 @@ sub run {
     # Now we add the incident repositories and do a zypper patch
     add_test_repositories;
     record_info('Updates', script_output('zypper lu'));
-    my $ret = trup_call('up', timeout => 300, proceed_on_failure => 1);
+    my $ret = 0;
+    $ret = trup_call('up', timeout => 300, proceed_on_failure => 1) unless check_var('DISABLE_UPDATE_WITH_PATCH', '1');
```

Comment:
That should just be

```suggestion
    $ret = trup_call('up', timeout => 300, proceed_on_failure => 1) unless get_var('DISABLE_UPDATE_WITH_PATCH');
```

It's good practice to not compare booleans for equality